src/zope/app/apidoc/utilities.rst

```diff
@@ -166,13 +166,15 @@ For interfaces we simply get
   >>> utilities.getPythonPath(ISample)
   'zope.app.apidoc.doctest.ISample'
 
-and for classes
+and for classes (note that we must pass a "bound method", because the
+unbound method is no longer present in Python 2):
 
   >>> class Sample(object):
   ...     def sample(self):
   ...         pass
 
-  >>> utilities.getPythonPath(Sample.sample)
+  >>> PY3 = str is not bytes
+  >>> utilities.getPythonPath(Sample().sample if PY3 else Sample.sample)
   'zope.app.apidoc.doctest.Sample'
```

Comment:
I think we should test both:

    >>> utilities.getPythonPath(Sample.sample)
    'zope.app.apidoc.doctest.Sample'
    >>> utilities.getPythonPath(Sample().sample)  # On Python 2 this is a bound method and needs special handling
    'zope.app.apidoc.doctest.Sample'